app/lib/features/home/widgets/quick_action_buttons.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:acter/common/themes/colors/color_scheme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/action_button_widget.dart';
+import 'package:acter/features/main/providers/main_providers.dart';
+import 'package:acter/features/tasks/sheets/create_update_task_list.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class QuickActionButtons extends ConsumerWidget {
+  const QuickActionButtons({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final lang = L10n.of(context);
+    return SingleChildScrollView(
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Wrap(
+            children: [
+              ActionButtonWidget(
```

Comment:
We are not checking whether any of these have actually spaces the user can post these to? We should check that and if we have nothing to show here, we probably want to fall back to the Create-Space & Create-Chat buttons in here.